metadata-picker.yml

```diff
@@ -1,6 +1,7 @@
 version: '3.3'
 
 services:
+  # fixme: Add auto-heal container similar to dev docker-compose.yml
```

Comment:
Why are you not doing this in the current branch? Other suggestion: can you use metadata-picker instead of docker-compose to have one file only?

Reply:
Thats why it's not fixed! I wasn't sure why there are two docker-compose files doing more or less the same thing!

Reply:
removed this file (together with a bunch of other no longer used docker related stuff)